packages/ui/src/components/ShipName.tsx

```diff
@@ -0,0 +1,64 @@
+import { cite } from '@urbit/aura';
+import { useMemo } from 'react';
+import { FontSizeTokens, FontWeightTokens, SizableText } from 'tamagui';
+
+import { useCalm } from '../contexts/calm';
+import { useContact } from '../contexts/contacts';
+
+export default function ShipName({
```

Comment:
Would it make sense to call this `AuthorName`? or `ContactName`? Just trying to keep the vocab consistent.

Reply:
ContactName probably works.